packages/xstate-graph/src/graph.ts

```diff
@@ -369,10 +378,13 @@ export function getPathFromEvents<
 
   const adjacency = performDepthFirstTraversal(behavior, optionsWithDefaults);
 
-  const stateMap = new Map<string, TState>();
+  const stateMap = new Map<SerializedState, TState>();
   const path: Steps<TState, TEvent> = [];
 
-  const initialStateSerial = serializeState(behavior.initialState, null);
+  const initialStateSerial = serializeState(
+    behavior.initialState,
+    undefined
+  ) as SerializedState;
```

Comment:
To match the style at the very similar call site in this file:
```suggestion
  const initialSerializedState = serializeState(
    behavior.initialState,
    undefined,
    undefined
  ) as SerializedState;
```

Reply:
I guess the style is just different at various places here - I personally would prefer a full name over a "serial" abbreviation. Especially since serial is a meaningful word and not just an abbreviation so while initially reading through this I was briefly confused about its meaning

Reply:
The serialization is not just an abbreviation; it is both a string representation of the traversed node (not necessarily the state, need to rename in the future) and is the main mechanism for the algorithms knowing whether a node has been visited or not.